docs/en/sql-reference/aggregate-functions/reference/anylast_respect_nulls.md

```diff
@@ -13,6 +13,8 @@ Selects the last value encountered, irregardless of whether it is `NULL` or not.
 anyLast_respect_nulls(column)
 ```
 
+Alias: `last_value_respect_nulls`.
```

Comment:
This is just internal syntactic sugar for `LAST VALUE RESPECT NULLS`. Not sure if we really need to document it.

Reply:
@Algunenano i'll modify the PR not to add that and also to remove these in that case? https://github.com/ClickHouse/ClickHouse/blob/27d7fd05a98016175dde4257189bcb851ab48dae/docs/en/sql-reference/aggregate-functions/reference/any_respect_nulls.md?plain=1#L10

Reply:
In general I would not document the `*_respect_nulls` (BTW there is a typo in `first_value_repect_nulls`) directly and instead mention the `RESPECT NULLS` modifier in `any` / `anyLast`, etc.

Reply:
@Algunenano while on the topic of syntactic sugar, there are some documentation issues for these 'missing' functions:

[or](https://github.com/ClickHouse/clickhouse-docs/issues/1993)
[not*](https://github.com/ClickHouse/clickhouse-docs/issues/1990)
[joinGetOrNull](https://github.com/ClickHouse/clickhouse-docs/issues/1968)
[global*](https://github.com/ClickHouse/clickhouse-docs/issues/1960)
[and](https://github.com/ClickHouse/clickhouse-docs/issues/1931)

Could you confirm that these are also just used internally, and not meant for direct use by the user?

Reply:
* or() -> I'd document it since it allows multiple arguments (e.g `or(x1, x2, x3, x4, x5, x6)`)
* and() -> Same as or
* not* -> These are syntactic sugar. No need to document them.
* joinGetOrNull: Not syntactic sugar, it should be documented alongside joinGet.
* global*: Syntactic sugar, except `globalVariable`, which is a completely different function that has nothing to do with `globalIn` family.